ui/src/main/webapp/src/app/components/reports/application-details/application-details.service.ts

```diff
@@ -0,0 +1,107 @@
+import {Injectable} from "@angular/core";
+import {Http} from "@angular/http";
+import {Observable} from "rxjs";
+import {AbstractService} from "../../../services/abtract.service";
+import {Constants} from "../../../constants";
+import {ApplicationDetailsDTO} from "windup-services";
+import {ProjectTraversalReducedDTO} from "windup-services";
+import {FileReducedDTO} from "windup-services";
+import {HintReducedDTO} from "windup-services";
+import {ClassificationReducedDTO} from "windup-services";
+import {TagReducedDTO} from "windup-services";
+
+@Injectable()
+export class ApplicationDetailsService extends AbstractService {
+
+    constructor(private _http: Http) {
+        super();
+    }
+
+    getApplicationDetailsData(executionId: number): Observable<ApplicationDetailsFullDTO> {
+        let url = `${Constants.GRAPH_REST_BASE}/graph/application-details/${executionId}`;
+
+        return this._http.get(url)
+            .map(res => res.json())
+            .map(res => <ApplicationDetailsDTO>res)
```

Comment:
I think this line is not needed, you could use .map((res: ApplicationDetailsDTO) => {}) and merge it with line below.